src/jsx-global.ts

```diff
@@ -5,6 +5,7 @@ declare global {
    * opt-in jsx intrinsic global interfaces
    * see: https://www.typescriptlang.org/docs/handbook/jsx.html#type-checking
    */
+  // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
In a `.d.ts` file it doesn't complain. Perhaps we are not linting those at all? In any case, just ignoring this error seems to be the right thing to do.

Reply:
I'm guessing namespaces might be allowed in .d.ts but not in .ts